package.json

```diff
@@ -30,6 +30,7 @@
         "@fortawesome/free-solid-svg-icons": "6.1.2",
         "@ls1intum/apollon": "2.11.2",
         "@ng-bootstrap/ng-bootstrap": "13.0.0",
+        "@ng-select/ng-select": "9.0.2",
```

Comment:
do we really need that? we would like to avoid adding new dependencies if this is not 100% necessary!!!

Reply:
@krusche You are right. I found out the same can be achieved with the ngBootstrap we already have as a dependency. Removed the dependency in 7a96dfa411746e4a9daa64f7f42f750fd90957f9